### ISSUE TITLE
Fire onAudioPositionChanged callback even when Audio is Paused

### ISSUE THREAD

**Issue by ayush221b** [rxlabz/audioplayer#84]
I am using a slider to seek() to a current position in the audio stream. Everything works fine when the AudioPlayer is Playing, but when it is Paused, the seek() works, but we don not get back the new position in the onAudioPositionChanged callback because it does not fire. due to this, the slider is not working. 

Can someone suggest a workaround for this?

**Comment by AnmolShahid** [comment]
Yes. I am facing the same issue now. It was working fine before. Did you get any solution?
